### DISCUSSION TITLE
Loading markdown from a url react-markdown

### DISCUSSION THREAD

**Discussion by amitojsingh366** [remarkjs/remark#633] [Q&A] [answered]
Is it possible to load in markdown from a url that links directly to a markdown file in react-markdown?

**Comment by wooorm** [comment, marked as answer]
No. Loading files is a complex topic that has many good solutions. In a browser, [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) can be used to get data from URLs
